src/Image/getUrl.js

```diff
@@ -0,0 +1,27 @@
+const cloudinaryId = 'buildo';
+const cloudinaryBaseUrl = `https://res.cloudinary.com/${cloudinaryId}/image/fetch/`;
+
+function isRetinaDisplay() {
+  if (window.matchMedia) {
+    const mq = window.matchMedia('only screen and (min--moz-device-pixel-ratio: 1.3), only screen and (-o-min-device-pixel-ratio: 2.6/2), only screen and (-webkit-min-device-pixel-ratio: 1.3), only screen  and (min-device-pixel-ratio: 1.3), only screen and (min-resolution: 1.3dppx)');
+    return (mq && mq.matches || (window.devicePixelRatio > 1));
+  }
+}
+
+function getUrl(src, { width: _width, height: _height, quality: _quality, format: _format }) {
```

Comment:
I would add `= {}` to the second parameters since all the inner props are optional